src/main/kotlin/org/jetbrains/research/transformations/Main.kt

```diff
@@ -1,5 +1,48 @@
 package org.jetbrains.research.transformations
```

Comment:
Just to clarify
Usually, the project structure looks as follows:
**-src** (means that it contains some **source** code)
**----main** (a directory that contains your **main** code)
**----test** (a directory that contains **test** code to prove that your main code works as expected)

So the rules are simple: everything that relates to the code in the **main** should be in the **main**. Usually, they are divided into source code and resources, so it becomes:

**-main**
**----kotlin** (means that it contains some source code on **kotlin**)
**----resources** (some additional files, for example, images, .csv or txt files, etc, that you use in your **kotlin** directory)

The same thing is going on with tests: they are divided into 2 folders: kotlin and resources. So, any code that you use for tests (setupParser, for example) should be in the **test** folder. Any resources that you use for tests  (pythonparser, for example) should be in the **test** folder. 

The last thing about the project structure is about **packaging**. You have added org.jetbrains.research.transformations into the main directory, that's great! (btw, [there](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) are official naming conventions for everything, including packages, take a look, it can be useful) However, this package structure should be repeated in the resources folder. So for data.csv, that you use in Main.kt it should be like this:

**-main**
**----kotlin**
**------org
--------jetbrains
----------research
------------transformations
--------------Main.kt**
**----resources** 
**------org
--------jetbrains
----------research
------------transformations
--------------data.csv**

This little thing allows you to easily get resources just by its name! So in Main.kt you can just write:
`val data = File(javaClass.getResource("data.csv))`


So please, change your project structure according to this information and use the correct way to store and get resources :)